src/index.ts

```diff
@@ -47,22 +52,21 @@ import { CoinType, HelloSwap } from "./helloSwap";
     process.on("SIGUSR2", exitHandler);
 })();
 
-// TODO: Funding should be handled by the create-comit-app
 async function startApp(whoAmI: string, index: number) {
-    const bitcoinWallet = new BitcoinWallet("regtest");
-    await bitcoinWallet.init(
+    const bitcoinWallet = await BitcoinWallet.newInstance(
+        "regtest",
         process.env.BITCOIN_P2P_URI!,
         process.env[`BITCOIN_HD_KEY_${index}`]!
     );
     await new Promise(r => setTimeout(r, 1000));
 
     const ethereumWallet = new EthereumWallet(
-        process.env[`ETHEREUM_HD_KEY_${index}`]!,
+        process.env[`ETHEREUM_KEY_${index}`]!,
         process.env.ETHEREUM_NODE_HTTP_URL!
     );
 
     const app = new HelloSwap(
-        process.env[`HTTP_PORT_CND_${index}`]!,
+        `http://localhost:${process.env[`HTTP_PORT_CND_${index}`]!}`,
```

Comment:
:question: What would we lose by using a `HTTP_URL_CND_*` instead of this?

Reply:
This actually needs to be correct, the `.env` now provides the full url.

Reply:
Done.